lib/mutant/mutator/node/case.rb

```diff
@@ -33,6 +33,8 @@ def emit_else_mutations
           else_branch = children.last
           else_index = children.length - 1
           return unless else_branch
+          return if else_branch.to_s.start_with?("(send\n  (const nil :T) :absurd")
```

Comment:
I'm looking for the proper way to do this.

Reply:
match the exact node: `else_branch.eql?(s(:send, (:const, nil, :T), :absurd))`. Protip next time use:

```
bundle exec ruby-parse --emit-ruby -e 'T.absurd'
```

to discover the ruby syntax for a given AST. `s` is in scope here.